### DISCUSSION TITLE
Hot reload not working

### DISCUSSION THREAD

**Discussion by sudeep2003** [bagisto/bagisto#10145] [Q&A]
For custom theme bagisto, I tired after watching video how to do. But I did every step still there is not work localhost:5173 hot reload. Website not run on localhost:5173.

**Comment by amit-webkul** [comment]
Hello @sudeep2003 ,

What issues are you facing with Vite's hot reloading? Please provide more details about the error. You can include screenshots or videos for better context.

**Comment by suraj-webkul** [comment]
Hello @sudeep2003,

It seems you're trying to run the app locally using port 5173, which is the default server provided by Vite. When you access the URL, you should see this [screen](https://prnt.sc/XYJfxqtw25MP). Please click on the highlighted link and try hot reloading again. If the issue persists, kindly provide more information so we can assist you further.

Thanks.

**Reply by sudeep2003** [reply]
Yes I see this default screen.

Is there not load the front of the website which is seen in app_url?

**Comment by suraj-webkul** [comment]
Hello @sudeep2003,

Please provide a screenshot of your .env file and let us know which URL you are using to run the application. Alternatively, you can provide additional information on how you've set up your application.

Thanks.

**Reply by sudeep2003** [reply]
This is my screenshot of .env

**Reply by sudeep2003** [reply]
https://prnt.sc/tEVHVjuB-Va6

**Comment by sudeep2003** [comment]
Also there is also an issue I notice in bagisto2.0 after new package update.
Unlinking Storage and linking again is also not working.

Image not load in admin panel in local.

I think this is issue about security.

Here is the screenshot:
https://prnt.sc/0bcwrSPTQTmx

**Reply by devansh-webkul** [reply]
Hello,

There might be some issue with your instance, i have verified in all the versions and its working fine.
